tool/grind.dart

```diff
@@ -33,17 +35,26 @@ updateRouteDart() {
     'git@github.com:jcollins-g/route.dart.git',
     _routeDir.path
   ]);
+  run('git', arguments: [
+    'clone',
+    '--branch',
+    'dart2-stable',
+    '--depth=1',
+    'git@github.com:jcollins-g/haikunatordart.git',
+    _haikunatorDir.path
+  ]);
   run('rm', workingDirectory: _routeDir.path, arguments: ['-rf', '.git']);
+  run('rm', workingDirectory: _haikunatorDir.path, arguments: ['-rf', '.git']);
 }
 
 @Task()
 analyze() {
-  new PubApp.global('tuneup')..run(['check']);
+  new PubApp.local('tuneup')..run(['check']);
 }
 
 @Task('Analyze the source code with the ddc compiler')
 ddc() {
-  PubApp ddc = new PubApp.global('dev_compiler');
+  PubApp ddc = new PubApp.local('dev_compiler');
```

Comment:
note: I think the `ddc` task is out of date and can be removed

Reply:
done